### PR TITLE
test(e2e,playwright): OfflineWarning

### PR DESCRIPTION
**Pull request by moradelboca**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705 OfflineWarning Component

Draft PR as required to help with coding.

### CODE REVIEW

**Comment by Amit-Morade** [comment]
Can I work on this?

**Comment by moradelboca** [comment]
> Can I work on this?

Of course! Idk if I have to give you permissions or something like that in order to contribute to this PR. Any clue?

**Comment by moradelboca** [comment]
> Hey @moradelboca, I noticed that this PR is still a draft. Is there anything we can help you with to get the PR ready for review?

I don't know what am I doing wrong. I created this PR to see if anyone had any idea.. a mod told me to do that in the ds server, but seems this isn't a good solution.
Any clue?

File: client/src/components/OfflineWarning/offline-warning.tsx [hunk 1]
```diff
@@ -42,10 +41,10 @@ function OfflineWarning({
   }
 
   return showWarning ? (
-    <>
+    <div data-playwright-test-label='offline-warning'>
       <div className='offline-warning alert-info'>{message}</div>
```
Comment (Ahmadkashif) [inline client/src/components/OfflineWarning/offline-warning.tsx:45]:
```suggestion
       <div className='offline-warning alert-info'  data-playwright-test-label='offline-warning' >{message}</div>
```
We don't wanna add a specific div for identifying the element.

File: client/src/components/OfflineWarning/offline-warning.tsx [hunk 2]
```diff
@@ -42,10 +41,10 @@ function OfflineWarning({
   }
 
   return showWarning ? (
-    <>
+    <div data-playwright-test-label='offline-warning'>
```
Comment (Ahmadkashif) [inline client/src/components/OfflineWarning/offline-warning.tsx:44]:
```suggestion
```

File: client/src/components/OfflineWarning/offline-warning.tsx [hunk 3]
```diff
@@ -42,10 +41,10 @@ function OfflineWarning({
...
       <div className='offline-warning alert-info'>{message}</div>
       <div style={{ height: `38px` }} />
-    </>
+    </div>
```
Comment (Ahmadkashif) [inline client/src/components/OfflineWarning/offline-warning.tsx:47]:
```suggestion
```

**Review by Ahmadkashif — CHANGES_REQUESTED** [review]
Hi @moradelboca 

Thank you for the PR. 

The error logs suggest that the first try to fetch the id for ofline-warning is failing. 

Can you try and fetch the id using our prescribed way? 
You can find the guidelines under the "how to contribute playwright tests" section on our docs.

**Comment by rahulsuresh-git** [comment]
@Ahmadkashif @moradelboca Is this still open?

**Pull request closed without merging**